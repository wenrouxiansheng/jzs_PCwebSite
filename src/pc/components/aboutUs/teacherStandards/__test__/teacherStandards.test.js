import React from "react";
import renderer from "react-test-renderer";
import TeacherStandards from "../index";

describe("PC:<TeacherStandards />", () => {
  it("render", () => {
    const wrapper = renderer.create(
      <TeacherStandards style={{ color: "red" }} />
    );
    let tree = wrapper.toJSON();

    //渲染测试
    expect(tree).toMatchSnapshot();
  });
});
