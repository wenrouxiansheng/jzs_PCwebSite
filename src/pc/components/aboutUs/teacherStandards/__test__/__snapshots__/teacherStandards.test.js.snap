// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PC:<TeacherStandards /> render 1`] = `
<div
  className="teacherStandards"
  style={Object {}}
>
  <div>
    <div
      className="img"
    >
      <img
        alt=""
      />
    </div>
    <h1>
      艺术培训综合体
    </h1>
    <p
      dangerouslySetInnerHTML={
        Object {
          "__html": "声乐、器乐、舞蹈、美术齐头并进，培养青少年综合素养",
        }
      }
    />
  </div>
  <div>
    <div
      className="img"
    >
      <img
        alt=""
      />
    </div>
    <h1>
      雄厚的师资力量
    </h1>
    <p
      dangerouslySetInnerHTML={
        Object {
          "__html": "数百名专职教师，毕业于国内外艺术院校",
        }
      }
    />
  </div>
  <div>
    <div
      className="img"
    >
      <img
        alt=""
      />
    </div>
    <h1>
      教研团队建设
    </h1>
    <p
      dangerouslySetInnerHTML={
        Object {
          "__html": "艺术培训行业科学、系统、完善的课程研发团队",
        }
      }
    />
  </div>
  <div>
    <div
      className="img"
    >
      <img
        alt=""
      />
    </div>
    <h1>
      量身提供教学计划
    </h1>
    <p
      dangerouslySetInnerHTML={
        Object {
          "__html": "只属于你的课程计划，“一位学员、一份计划、一种方法”",
        }
      }
    />
  </div>
</div>
`;
